src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -81,75 +79,74 @@ public string GetDumpFile()
         }
 
         /// <inheritdoc/>
-        public void StartProcessDump(int processId, string dumpFileGuid, string testResultsDirectory, bool isFullDump = false)
+        public void StartTriggerBasedProcessDump(int processId, string dumpFileGuid, string testResultsDirectory, bool isFullDump = false)
```

Comment:
>StartTriggerBasedProcessDump [](start = 20, length = 28)

Duplicate code again in StartTriggerBasedProcessDump, StartHangBasedProcessDump , lets try to clean it.